### PR TITLE
Setting app authentification with Devise Gem

### PR DESCRIPTION
**Pull request by chelobotix**
In this PR we fulfilled the following tasks:
- [x] Remove current_user method in ApplicationController
- [x] Install and setup devise
- [x] register a new user.
- [x] User logs in with a combination of email and password.
- [x] Hashed password should be stored in the database.
- [x] Ask for confirmation of email.
- [x] Can reset password.
- [x] Modify the views in devise

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  </div>
+<% end %>
+
+<div class="messages">
+<<<<<<< Updated upstream
+  <%= render "devise/shared/links" %>
+=======
+  <%= render "devise/shared/links" %>
+>>>>>>> Stashed changes
+</div>
```
Comment (cynthiainga) [inline app/views/devise/sessions/new.html.erb]: - [x] When I run your app I get an error and I can't check if a user can log in or signup, please kindly fix the error and make sure your devise is setup correctly 😁 
- [x] As you can see there are two ` <%= render "devise/shared/links" %>` you should accept the current change, and there is a space at the end of `...links" %>` it should be `...links"%>`. Kindly fix the space error in the `registrations/new.html.erb` file and everything will be fine 😁 

Screenshot 2023-05-24 at 22 19 43

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @chelobotix, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (chelobotix) [reply app/views/devise/sessions/new.html.erb]: @cynthiainga thank you for your observations. I didn't notice that duplicate link

**Review by brytebee — APPROVED** [review]
Hi, @chelobotix & @luis-pomare 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great DB setup.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**